apps/cnquery/cmd/sbom.go

```diff
@@ -100,13 +110,27 @@ var sbomCmdRun = func(cmd *cobra.Command, runtime *providers.Runtime, cliRes *pl
 		log.Fatal().Err(err).Msg("failed to get exporter for output format: " + output)
 	}
 
-	for _, bom := range boms {
+	outputTarget := viper.GetString("output-target")
+	for i := range boms {
+		bom := boms[i]
 		output := bytes.Buffer{}
 		err := exporter.Render(&output, &bom)
 		if err != nil {
 			log.Fatal().Err(err).Msg("failed to render SBOM")
 		}
-		fmt.Println(output.String())
+
+		if outputTarget != "" {
+			filename := outputTarget
+			if len(boms) > 1 {
+				filename = fmt.Sprintf("%s-%d.%s", path.Base(outputTarget), i, path.Ext(outputTarget))
```

Comment:
nit: this might be a problem in CI/CD setups, a user wouldn't expect that we change the filename. can we not write all boms to one file?

Reply:
Mmh, the current format only has one report included, so we would need one file for a sbom.